src/views/project/publishPartition/PublishPartitionView.vue

```diff
@@ -0,0 +1,95 @@
+<script setup>
+import axios from 'axios'
+import router from '@/router'
+import { ref, onMounted } from 'vue'
+import { useRoute } from 'vue-router'
+import LoadingIndicator from '@/components/project/LoadingIndicator.vue'
+
+const route = useRoute()
+const projectId = route.params.id
+const isLoaded = ref(false)
+let partitions = ref([])
+
+onMounted(() => {
+  getPartitions()
+  isLoaded.value = true
```

Comment:
This doesn't wait for the partitions to be done before setting `isLoaded`. It gets set immediately. Instead you should change the function to be async and await for `getPartitions` to be loaded.

```
onMounted(async () => {
  await getPartitions()
  isLoaded.value = true
}
```